src/mito2/src/error.rs

```diff
@@ -533,6 +533,13 @@ pub enum Error {
         error: std::io::Error,
         location: Location,
     },
+
+    #[snafu(display(
+        "Verify checksum failed! calculated: {}, expected: {}",
+        calculated,
+        expect
+    ))]
+    VerifyChecksum { calculated: String, expect: String },
```

Comment:
Consider adding a location.
```suggestion
    VerifyChecksum {
        calculated: String,
        expect: String,
        location: Location,
    },
```